src/recordpage.cpp

```diff
@@ -352,6 +349,13 @@ RecordPage::RecordPage(QWidget* parent)
         ui->viewPerfRecordResultsButton->setEnabled(false);
     });
 
+    connect(m_perfRecord, &PerfRecord::debuggeeCrashed, this, [this]{
+        ui->applicationRecordWarningMessage->setText(tr("Debugge crashed. Results may be unusable."));
+        ui->applicationRecordWarningMessage->setVisible(true);
+
+        QTimer::singleShot(3000, this, [this]{ui->applicationRecordWarningMessage->setVisible(false);});
```

Comment:
please don't auto hide messages. the user can hide it manually if desired. We should only clear/hide the message manually when the user re-starts a recording